Enigmatry.Entry.Blueprint.Api.Tests/CoreFeatures/ApiSetupFixture.cs

```diff
@@ -85,9 +100,23 @@ public async Task TestCreateReturnsValidationErrors(string name,
         string validationField,
         string validationErrorMessage)
     {
-        var command = new CreateOrUpdateUser.Command { FullName = name, EmailAddress = emailAddress };
-        var response = await Client.PostAsJsonAsync("users", command);
+        var command = new CreateOrUpdateUser.Command
+        {
+            Id = null,
+            FullName = name,
+            EmailAddress = emailAddress,
+            RoleId = Role.SystemAdminRoleId,
+            UserStatusId = UserStatusId.Inactive
+        };
+        var response = await Client.PostAsJsonAsync("users", command, HttpSerializationOptions.Options);
 
         response.Should().BeBadRequest().And.ContainValidationError(validationField, validationErrorMessage);
     }
+
+    [Test]
+    public void TestAutoMapperMappings()
```

Comment:
I have noticed that there was a problem with Automapper mappings (ProductController). I have added test that checks mappings.